From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 17 Nov 2015 18:41:38 +0000
Subject: ehci: Fix ABI change in 3.16.7-ckt19
Forwarded: not-needed

Move the new bitfield in struct ehci_hcd after the other bits, to
avoid an ABI change.  I doubt this is going to be used OOT but it's
easy enough to do.

---
--- a/drivers/usb/host/ehci.h
+++ b/drivers/usb/host/ehci.h
@@ -215,7 +215,6 @@ struct ehci_hcd {			/* one per controlle
 	/* SILICON QUIRKS */
 	unsigned		no_selective_suspend:1;
 	unsigned		has_fsl_port_bug:1; /* FreeScale */
-	unsigned		has_fsl_hs_errata:1;	/* Freescale HS quirk */
 	unsigned		big_endian_mmio:1;
 	unsigned		big_endian_desc:1;
 	unsigned		big_endian_capbase:1;
@@ -227,6 +226,10 @@ struct ehci_hcd {			/* one per controlle
 	unsigned		frame_index_bug:1; /* MosChip (AKA NetMos) */
 	unsigned		need_oc_pp_cycle:1; /* MPC834X port power */
 	unsigned		imx28_write_fix:1; /* For Freescale i.MX28 */
+#ifndef __GENKSYMS__
+	unsigned		has_fsl_hs_errata:1;	/* Freescale HS quirk */
+	/* 18 bits spare */
+#endif
 
 	/* required for usb32 quirk */
 	#define OHCI_CTRL_HCFS          (3 << 6)
